HACKING.md

```diff
@@ -106,6 +106,46 @@ otherwise an official image can be run with
 
 FAF should be available at ```http://localhost:8080/faf/```
 
+### Reporting into FAF
+1. Set up your server URL for libreport
+
+    Insert `URL = http://localhost:8080/faf` into `/etc/libreport/plugins/ureport.conf`
+2. Create crash
+
+    Install package `will-crash` and then execute `will_abort`
```

Comment:
I would mention a `abrt-auto-reporting enabled` to enable automatic sending of uReports.